packages/rum/test/benchmarks/run.js

```diff
@@ -46,31 +47,34 @@ const REPORTS_DIR = join(__dirname, '../../reports')
     await customApmBuild(filename)
 
     const server = await startServer(filename)
+    const close = promisify(server.close.bind(server))
     /**
      * object cache holding the metrics accumlated in each run and
      * helps in processing the overall results
      */
     const resultMap = new Map()
 
     for (let scenario of scenarios) {
-      const browser = await launchBrowser()
-      const url = `http://localhost:${port}/${scenario}`
-      const version = await browser.version()
-
-      /**
-       * Add common set of metrics for all scenarios
-       */
-      resultMap.set(scenario, getCommonFields({ version, url, scenario }))
-
-      for (let i = 0; i < runs; i++) {
-        const metrics = await gatherRawMetrics(browser, url)
-        Object.assign(metrics, { scenario, url })
-        await analyzeMetrics(metrics, resultMap)
+      for (let type of browserTypes) {
+        const url = `http://localhost:${port}/${scenario}`
+        /**
+         * Add common set of metrics for all scenarios
+         */
+        const key = `${scenario}.${type}`
```

Comment:
It seems to me that we should put the type first and then scenario, WDYT?

Reply:
IMO, browser is just a parameter. scenario is the top level and ever growing and tunable list. Like how its done in Karma. `name of the suite: { browser: "chrome"}`